projects/packages/paypal-payments/src/block/utils.js

```diff
@@ -0,0 +1,62 @@
+import { trimEnd } from 'lodash';
+import { getCurrencyDefaults } from '../../shared/currencies';
```

Comment:
You would need to bring this in too.

Reply:
I forgot to comment on this one yesterday, it was addressed in [9e281ea](https://github.com/Automattic/jetpack/pull/43413/commits/9e281ea750dccb6b0f1bb46ca92d05d458428e8e).